File: Programmers/자릿수 더하기.js
// https://programmers.co.kr/learn/courses/30/lessons/12931

function solution(n) {
  return n
    .toString()
    .split("")
    .reduce((acc, cur) => acc + parseInt(cur), 0);
}

const n = 123;
console.log(solution(n));

File: Programmers/자릿수 더하기.swift
// https://programmers.co.kr/learn/courses/30/lessons/12931?language=swift
// 자릿수 더하기

import Foundation

func solution(_ n:Int) -> Int {
    var answer: Int = 0
    var temp: Int = n
    
    while temp > 0 {
        answer += temp % 10
        temp = Int(floor(Double(temp) / 10))
    }
    
    return answer
}

let n: Int = 123	
// 6

// let n: Int = 987	
// // 24

print(solution(n))
